pkgs/development/compilers/ldc/generic.nix

```diff
@@ -0,0 +1,164 @@
+{ bootstrapVersion ? false
```

Comment:
bootstrapVersion can be removed now.
This makes this derivation much more streamlined as many optionals can be inlined and some can be removed completely.

Reply:
removed.